sdk/data/azcosmos/emulator_aad_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+)
+
+func TestAAD(t *testing.T) {
```

Comment:
Not yet added any recorded test to this SDK, unsure yet how to approach to doing them

Reply:
The added Cosmos DB emulator live test helps achieve the goal of testing against an AAD like JWT token. Since the Cosmos client is mostly doing a pass through for the obtained AAD token, there's not much else to test here.

Reply:
Regarding recorded live tests, if you choose to pursue them in the future, the docs are [here](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/internal/recording).